src/rover/Makefile

```diff
@@ -0,0 +1,15 @@
+SOURCES=main.cpp camera.cpp
+HEADERS=camera.hpp
+LFLAGS=-L../../bin -lnetwork
+CFLAGS=-Wall -Wextra -Wpedantic -std=c++11 -g
+EXECUTABLE=rover
```

Comment:
I would like the executable to be built in `/bin/`, which would make the `EXECUTABLE` variable `../../bin`.